### PR TITLE
Considerations about DNS nameservers

### PR DESCRIPTION
**Pull request by lzap**
Users are constantly running into this issue upstream.

### CODE REVIEW

**Comment by melcorr** [comment]
Hi @lzap, should this be wrapped for upstream only if this is a problem that happens upstream?

**Comment by lzap** [comment]
This is valid for all deployment types.

File: guides/doc-Provisioning_Guide/topics/Networking.adoc [hunk 1]
```diff
@@ -206,6 +206,20 @@ IPTABLES_MODULES="ip_conntrack_tftp"
 .Checking for Existing Domains
 {ProjectServer} might already have the relevant domain created as part of {ProjectServer} installation. Switch the context to `Any Organization` and `Any Location` then check the domain list to see if it exists.
 
+.Considerations when configuring DNS servers
+
+During DNS record creation, {Project} performs conflict DNS lookups to verify that hostname is not in active use. This check will be performed against one of the following DNS servers:
```
Comment (ekohl) [inline guides/doc-Provisioning_Guide/topics/Networking.adoc]:
```suggestion
During DNS record creation, {Project} performs conflict DNS lookups to verify that the hostname is not in active use. This check will be performed against one of the following DNS servers:
```

File: guides/doc-Provisioning_Guide/topics/Networking.adoc [hunk 2]
```diff
@@ -206,6 +206,20 @@ IPTABLES_MODULES="ip_conntrack_tftp"
...
+
+* system-wide resolver if *Adminster* > *Settings* > *Query local nameservers*
```
Comment (ekohl) [inline guides/doc-Provisioning_Guide/topics/Networking.adoc]: Probably best to add something like `is set to true`

**Comment by lzap** [comment]
All remarks done.

**Pull request merged**